### PR TITLE
Turkish Language Translations

### PR DESCRIPTION
**Pull request by guneysus**
## Description
Adding Turkish language translations.

## Types of changes
- [x] Actions
- [x] About
- [x] Daterange ricker
- [x] Exceptions
- [x] Flash messages
- [x] Invoice calculator
- [x] Invoice number generator
- [x] Invoice renderer
- [x] Messages
- [x] Pager fanta
- [x] Plugins
- [x] System configuration
- [x] Tags
- [x] Teams
- [x] Validators

## Checklist
- [ ] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [ ] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Hi @guneysus ,
thanks for your contribution 👍 
Can you let me know, once you think its done from your side, then I will have a look.

**Comment by kevinpapst** [comment]
Oh and I guess you already saw that, but this link might be useful:
https://www.kimai.org/documentation/translations.html#adding-a-new-language
Especially the config changes are important.

**Comment by guneysus** [comment]
Hi @kevinpapst.
Thanks for your guidance.

Completed the steps for adding a new language. Import frontend local in Turkish both select2 and moment.js.

I suspect one or two wrong translations.

**Comment by kevinpapst** [comment]
Ok, then I will compile the frontend assets now,  push them to your branch and install it on a test domain.

**Comment by kevinpapst** [comment]
Can you please pull my changes before you continue?
Here is the demo installation (including these 2 commits): https://demo-branch.kimai.org/tr/

As you can see the login pages are not yet translated.
They are coming [from another package](https://github.com/kevinpapst/AdminLTEBundle), may I ask you to translate these one as well?
Its much easier than Kimai, just [one file per language](https://github.com/kevinpapst/AdminLTEBundle/tree/master/Resources/translations).

**Comment by kevinpapst** [comment]
Reinstalled the demo with the new AdminLTE Bundle: https://demo-branch.kimai.org/tr/
Can you test everything and let me know once you think this PR is ready for merge?!

**Comment by guneysus** [comment]
@kevinpapst what is the demo user/pass?
Login, reset passwords and New account pages look good.

**Comment by kevinpapst** [comment]
See https://www.kimai.org/demo/

**Comment by guneysus** [comment]
There is some Swedish text on page: https://demo-branch.kimai.org/tr/calendar/ (Idag, Månad etc)

**Comment by kevinpapst** [comment]
Ah, forgot to add Turkish for the calendar frontend. 
Fixed and re-deployed.

**Comment by guneysus** [comment]
Logged in susan_super and john_user. Admin and user pages looks good to me.
Some sentence translations may be improved. But as as user I can not see any weird translations.
I can not translated yet `Number of running time entries from which a warning is displayed` 😞

File: translations/system-configuration.tr.xliff [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                <source>label.timesheet.rules.allow_future_times</source>
+                <target state="translated">Allow time entries in the future</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.active_entries.hard_limit">
+                <source>label.timesheet.active_entries.hard_limit</source>
+                <target state="translated">İzin verilen eşzamanlı zaman kaydı sayısı</target>
+            </trans-unit>
+            <trans-unit id="label.timesheet.active_entries.soft_limit">
+                <source>label.timesheet.active_entries.soft_limit</source>
+                <target state="translated">Number of running time entries from which a warning is displayed</target>
```
Comment (kevinpapst) [inline translations/system-configuration.tr.xliff]: You mean this one?

Comment (kevinpapst) [reply translations/system-configuration.tr.xliff]: Behause you don't understand the meaning? You are right, that could be improved...
It means: `Number of running timesheet records before a warning is displayed`.

Comment (guneysus) [reply translations/system-configuration.tr.xliff]: I understand but can not find a meaningful translation. Let me think.

Comment (kevinpapst) [reply translations/system-configuration.tr.xliff]: So, when you set this number to 3 and have two running timesheet records everything is good. But if you start the 3 record, you will see a little warning:
Bildschirmfoto 2019-10-15 um 15 35 25
In this example only 1 record is allowed

**Pull request merged**